### ISSUE TITLE
Enabling automatic safari zone disables the RTC and other enabled auto* scripts and does not function

### ISSUE THREAD

**Issue by ItsDeidara** [Ephenia/Pokeclicker-Scripts#364]
For some reason if I enabled the autosafarizone script any other auto scripts(like autohatchery or automine or simpleautofarmer) stop functioning as a whole, alongside any time-based items within the game(like automatic wild Pokemon battles, dungeon timers, harvest timers, battle item timers etc) 

On top of this, the autosafarizone script even when enabled by itself does not currently seem to function at all. Specifically, none of its toggles seem to appear to even use it.

**Comment by lnlyssg** [comment]
Yup, same issue here - auto clicker seems to work for a very short time but I need to re-click to start it again. Disabling the Safari script gets everything working once more.

**Comment by dragons-childe** [comment]
Hi, can confirm this issue is still taking place with the latest version (v.0.10.17) and the update installed, at least on the desktop client - enabling the auto safari doesn't disable all the other scripts for me, but it doesn't actually load the auto safari options. (Confirmed as of 12.01.2024 to be an open issue.)

**Comment by lnlyssg** [comment]
Bit late coming back to this but I ran the Script Fixer Upper which seemed to get everything working again (was definitely on the latest versions)

**Comment by dragons-childe** [comment]
Thanks for the suggestion, ran that and it now works perfectly again, thanks for the help!

**Comment by dragons-childe** [comment]
Hasn't fixed the other issue with the autoclicker, but at least the autosafari works again
